README.rst

```diff
@@ -140,7 +140,23 @@ Skipping Tests
 ~~~~~~~~~~~~~~
 
 Doctest provides the ``+SKIP`` directive for skipping statements that should
-not be executed when testing documentation. However, it is often useful to be
+not be executed when testing documentation.
+
+.. code-block:: python
+
+    >>> open('file.txt') # doctest: +SKIP
+
+And in `.rst` documentation, whole code example blocks can be skipped with the
+directive
+
+.. code-block:: rst
+
+    .. doctest-skip::
```

Comment:
It seems that there is no example for the conditional* one either, you might be want to add one as part of this PR, too.

* e.g. `.. doctest-requires:: scipy` before a codeblock that only runs with scipy. Or in fact this very example can be one with `asdf`